components/LayerObject.vue

```diff
@@ -42,9 +93,23 @@
         display: flex;
         flex-direction: column;
         margin-left: auto;
+        & :hover {
+            cursor: pointer;
+        }
     }
 
     .layer-text {
         margin: 0;
     }
+
+    .left-over {
+        background: linear-gradient(90deg, #6360f4 44.5%, #f460b7 100%);
```

Comment:
Please either use the already defined gradient or define a  new one, with more muted colors in the variables.

Reply:
Oh I did not see this, I think it would be best to make it the easiest for Alica to add a TODO. Also to make it clear that the class should be removed.

Reply:
The dragging of the list was not designed so I showed the option of having a 'ghost' placeholder but it is not really clear.